kubernetes/manifest.json

```
@@ -33,9 +33,11 @@
       "[kubernetes] Monitor Kubernetes Statefulset Replicas": "assets/monitors/monitor_statefulset_replicas.json"
     },
     "dashboards": {
-      "Kubernetes - Pods": "assets/dashboards/kubernetes_pods.json",
+      "Kubernetes Pods Overview": "assets/dashboards/kubernetes_pods.json",
```

```suggestion
      "Kubernetes - Pods": "assets/dashboards/kubernetes_pods.json",
```
Let's keep the key the same as before. It's not used for the dashboard title but will affect the backend dash db.

---

oh gotcha, sorry about the sneaky update :D